sourcegraph/sourcegraph
# Fix prometheus config permissions

The prometheus.yml and alertmanager.yml files need to be writable by the `sourcegraph` user to allow configuration to by synced from siteConfig to alertmanager/prometheus.

## Test plan



- Build image locally
- Check successful fix on k8s-dogfood
